src/main/java/seedu/duke/Duke.java

```diff
@@ -48,8 +49,13 @@ public void run() {
         ui.printWelcomeMessage();
         boolean isExit = false;
         storage.loadAll(data);
+
         while (!isExit) {
             try {
+                ui.printDividerLine();
+                Command reminder = currentParse.parse("reminder");
```

Comment:
Would you like to shift this command outside the while loop? will the reminder command keep reminding for every other command we write?

Reply:
It will not run for every command. I would prefer not to switch it out because there's a try catch